definitions/marts/looker_studio_marts/ecf/ls_ecf_provider_activity_partnership_changes.sqlx

```diff
@@ -0,0 +1,69 @@
+config {
+    type: "table",
+    bigquery: {
+        partitionBy: "event_date",
+        clusterBy: ["cpd_lead_provider_name"]
+    },
+    description: "This mart contains successful (200 status code) API requests made by a Lead Provider on the API path containing 'partnerships/ecf' to change a partnership's associated delivery partner. \n \
+	This mart includes changes where the occurred_at date is in the last 180 days only. \n \
+	The purpose of this mart is to support ECF Contract Management in monitoring Lead Provider made changes, namely, the change of deliver partners associated with partnerships. \n \
+	The data within this mart will be used in the ECF Contract Management Dashboard to provide a high level count of changes made by the Lead Provider on an Event Date as well as key information that can assist in their conversations with LPs.",
+    columns: {
+      event_date: "The date the change was made in the service. This is called occurred_at in the source data table and renamed to event_date in the mart.",
+      event_date_delta: "The number of days between the event_date and the current date.",
+      cpd_lead_provider_name: "The name of the lead provider who performed the action. This is determined by the request_user_id which is captured in the service and mapped against the cpd_lead_provider_id in cpd_lead_providers_latest. This connection has been confirmed as correct by LPDOB.",
+	  partnership_id: "The ID of the created partnership.",
+	  school_urn: "The Unique Reference Number (URN) of the school involved in the newly created partnership with the lead provider.",
+	  cohort: "The cohort associated with the created partnership.",

```

Comment:
Similar question, is this reported by the LP?

Reply:
?

Reply:
This is set by the LP when creating the partnership. Field description has been updated.